src/main/java/com/fabriccommunity/spookytime/mixin/SkeletonPumpkinMixin.java

```diff
@@ -0,0 +1,21 @@
+package com.fabriccommunity.spookytime.mixin;
+
+import com.fabriccommunity.spookytime.MixinHelpers;
+import net.minecraft.entity.EquipmentSlot;
+import net.minecraft.entity.mob.AbstractSkeletonEntity;
+import net.minecraft.entity.mob.SkeletonEntity;
+import net.minecraft.item.ItemStack;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Redirect;
+
+/**
+ * @author Indigo Amann
+ */
+@Mixin(AbstractSkeletonEntity.class)
+public class SkeletonPumpkinMixin {
+    @Redirect(method = "initialize", at = @At(value = "INVOKE", target = "Lnet/minecraft/entity/mob/AbstractSkeletonEntity;getEquippedStack(Lnet/minecraft/entity/EquipmentSlot;)Lnet/minecraft/item/ItemStack;"))
+    private ItemStack getStack(AbstractSkeletonEntity entity, EquipmentSlot equipmentSlot_1) {
```

Comment:
Can we change the method name to `getEquippedStack` just for extra clarity?

Reply:
Well its specifcally for the pumpkin, sooo, I'll change it, but not to that

Reply:
Yes, just anything more descriptive works. Thanks!